### ISSUE TITLE
Overlay identifying the mountains

### ISSUE THREAD

**Issue by robin13** [radoondas/radoondas.io#2]
It would be neat to have an overlay which shows the names of the mountains and other significant points of interest visible in the picture.
Bonus: if the overlay can be turned on/off with a button on the page, but still allowing the underlying image to be saved with right-mouseclick.

**Comment by radoondas** [comment]
What a lovely idea! Thanks!
After a quick research and dirty test, it seems that simple CSS overlay would do the trick. Check the preview.

WDYT?

https://user-images.githubusercontent.com/8320348/122929121-db0d2d00-d36a-11eb-866a-0ebe16c039c1.mov

**Comment by robin13** [comment]
Yeah - that looks neat! Does it still allow a right-mouse-click on the image to download?

**Comment by radoondas** [comment]
No :) 
At least not in the current dirty implementation test.
Does anyone need that?

**Comment by robin13** [comment]
How will I otherwise share the shock when the mountains have disappeared (in the clouds) with my friends?! ;-P
